specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json

```diff
@@ -79,6 +79,55 @@
         }
       }
     },
+    "/keys/{key-name}/roll": {
```

Comment:
Nit: Method URIs have been pretty consistent. Should this maybe be "rotate"?

Reply:
...I see from below even the permission is called "rotate".

Reply:
Have changed to rotate now